dropshot/src/lib.rs

```diff
@@ -742,7 +742,7 @@ mod dtrace;
 mod api_description;
 mod body;
 mod config;
-mod error;
+pub mod error;
```

Comment:
Are you thinking users would use this as `dropshot::error::Something`?  We could organize stuff that way but it seems different from what we've done with everything else in Dropshot.

Reply:
I'm fine with changing this to be not-`pub` and reexporting everything at the root level!